API/Extensions/ApplicationServiceExtensions.cs

```diff
@@ -33,6 +34,13 @@ public static void AddApplicationServices(this IServiceCollection services, ICon
         services.AddScoped<ITaskScheduler, TaskScheduler>();
         services.AddScoped<ICacheService, CacheService>();
         services.AddScoped<IArchiveService, ArchiveService>();
+
+        services.AddScoped<IImageConverterProvider, JpegXLImageConverterProvider>();
```

Comment:
Since all providers are IImageConverterProviders. You can DI them as IEnumerable<IImageConverterProviders> in the ImageConveterServices, and you will get the whole collection of Dependency Injected Converters added in here.